src/components/shared/ArrowPanel.css

```diff
@@ -150,3 +156,39 @@
 .arrowPanelCancelButton:hover:active {
   background-image: linear-gradient(rgba(0,0,0,0.2), rgba(0,0,0,0.2));
 }
+
+.arrowPanelOpenMetainfo {
+  width: 350px;
+  font-size: 12px;
+  line-height: 20px;
+  padding-top: 5px;
+}
+
+.arrowPanelOpenMetainfo,
+.arrowPanelOpenMetainfo .arrowPanelArrow::before {
+  background-color: white;
+}
+
+.arrowPanelSection {
+  margin-top: 7px;
+  margin-bottom: 12px;
+}
+
+.metaInfoRow {
+  display: block;
+}
+
+.metaInfoLabel {
+  width: 120px;
+  color: var(--grey-40);
+  display: inline-block;
```

Comment:
inline-block is not a very reliable means of achieving column layout. The layout will break as soon as the contents of the right column wrap into multiple lines.